Source/FullScan/DetailedReplication.lua

```diff
@@ -0,0 +1,42 @@
+function Auctionator.FullScan.DetailedReplication()
+  local totalReplicateItems = C_AuctionHouse.GetNumReplicateItems()
```

Comment:
Please add whitespace. Code should read like an essay; intro, body (with paragraphs composed of related sentences/concepts), conclusion. Hard for me to parse without the whitespace.